src/VimeoUploadLambda/index.js

```diff
@@ -32,6 +32,10 @@ const uploadToVimeo = (bucket, video) => {
   );
 };
 
+const getCloudfrontName = (bucket) => {
+  `https://${bucket.replace('execonline-', '')}.execonline.com`
```

Comment:
1. Where does the `bucket` value ultimately come from?
2. Can it be trusted to not contain URL entities or be maliciously set to something that would cause the domain to be changed?
3. Is this lambda ever referenced by development which wouldn't to conform to this new URL structure?

Reply:
> Where does the bucket value ultimately come from? Can it be trusted to not contain URL entities or be maliciously set to something that would cause the domain to be changed?
This comes from an ENV variable, so can be trusted
> Is this lambda ever referenced by development which wouldn't to conform to this new URL structure?
Dev also uses the staging bucket for this lambda :/